reactive/etcd.py

```diff
@@ -329,6 +329,8 @@ def install_etcd():
     if channel:
         snap.install('etcd', channel=channel, classic=False)
 
+    set_state('snap.installed.etcd')
```

Comment:
I don't think this is necessary given that `snap.install('etcd', ...)` sets the same state here: https://github.com/stub42/layer-snap/blob/9f8b9a03f57b528d200d037789db910ca79698b9/lib/charms/layer/snap.py#L58

Reply:
Ah, thanks!